Firmware/Marlin_main.cpp

```diff
@@ -6189,6 +6188,7 @@ SERIAL_PROTOCOLPGM("\n\n");
         LCD_MESSAGERPGM(CAT4(CUSTOM_MENDEL_NAME,PSTR(" "),MSG_OFF,PSTR(".")));
         lcd_update(0);
 	  break;
+    #endif
```

Comment:
hmm, this may have had some reason to be enabled even without the PS_ON_PIN, don't you think?